yaml/pois.yaml

```diff
@@ -172,6 +172,14 @@ filters:
     min_zoom: 17
     output:
       kind: {col: tags->zoo}
+  - filter: {waterway: waterfall}
+    min_zoom: CASE WHEN to_number(tags->'height', '9999.99') < 10 THEN 15
```

Comment:
Re: `height` - when you imported your database, did you use the stock osm2pgsql `default.style` or did you use [the version in vector-datasource](https://github.com/mapzen/vector-datasource/blob/master/osm2pgsql.style#L98) which includes `height` as part of the table?

Re: logic in two places - I agree, that's why I think the best option is to have the logic in the YAML file only. Otherwise we'll update that logic at some point in the future and wonder why it's having no effect and have to go chasing through the other layers of the stack to debug it.